### ISSUE TITLE
Toggle: label for and input id

### ISSUE THREAD

**Issue by jjagielka** [themesberg/flowbite#219]
**Describe the bug**

From: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Label

> To associate the <label> with an <input> element, you need to give the <input> an id attribute. The <label> then needs a for attribute	whose value is the same as the input's id.
> Alternatively, you can nest the <input> directly inside the <label>, in which case the **for and id attributes are not needed** because the association is implicit:

```html
	<label>Do you like peas?
	<input type="checkbox" name="peas">
	</label>
```

Toggle example is implemented with the latter option i.e. input embedded in label, so `for` and `id` are redundant.

```svelte
<label for="default-toggle" class="inline-flex relative items-center cursor-pointer">
  <input type="checkbox" value="" id="default-toggle" class="sr-only peer">
  <div class="w-11 h-6 ..."></div>
  <span class="ml-3 text-sm font-medium text-gray-900 dark:text-gray-300">Toggle me</span>
</label>
```

**Comment by alekstrust** [comment]
Thanks for noticing.

I'm using the toggle inside a dropdown menu. Due to this `for` attribute, the popup was getting closed.

**Comment by zoltanszogyenyi** [comment]
Fixed in `v1.5.4` update with this commit https://github.com/themesberg/flowbite/commit/88237cf66bd1e6ce0267a23f82cba1fc908f11b6.
